guides/source/i18n.md

```diff
@@ -918,11 +918,11 @@ end
 The key for the error message in this case is `:blank`. Active Record will look up this key in the namespaces:
```

Comment:
I think the idea makes sense but now the example doesn't fit with this line introducing it. When `.[key]` is included the list is no longer a list of namespaces, its a list of the exact keys going to be checked

Reply:
Thank you for your advice. It makes sense. I updated with 6f56e373b788e7f5c81be6a9831cec27667916f2, and moved abstract list after `:blank` examples. Besides, I changed the wording.